logstash-core/lib/logstash/config/loader.rb

```diff
@@ -0,0 +1,90 @@
+require "logstash/config/defaults"
+
+module LogStash; module Config; module Loader
+  def self.format_config(config_path, config_string)
+    config_string = config_string.to_s
+    if config_path
+      # Append the config string.
+      # This allows users to provide both -f and -e flags. The combination
+      # is rare, but useful for debugging.
+      config_string = config_string + load_config(config_path)
+    else
+      # include a default stdin input if no inputs given
+      if config_string !~ /input *{/
+        config_string += LogStash::Config::Defaults.input
```

Comment:
My expectation would be see this line as something like  ```LogStash::Config::Defaults::INPUT``` does it makes sense to you? why using a method instead of a constant?